.mockery.yaml

```diff
@@ -43,6 +43,7 @@ packages:
       ProjectGetter:
       RbacEnforcer:
       SettingsGetter:
+      UserGetter:
```

Comment:
codegen might be missing for UserGetter ? 🤔

Reply:
This is just the way mockery expects to have the list interfaces provided. If there is no additional configs for that mock, we should just provide the name of the interface followed by a colon.